modules/BiomesAPI/build.gradle

```diff
@@ -69,11 +67,44 @@ for (dependency in moduleConfig.dependencies) {
 // Gradle uses the magic version variable when creating the jar name (unless explicitly set somewhere else I guess)
 version = moduleConfig.version
 
+import org.gradle.internal.logging.text.StyledTextOutputFactory
+import static org.gradle.internal.logging.text.StyledTextOutput.Style
+
+// The only case in which we make module non-snapshots is if release management is enabled and BRANCH_NAME is "master"
+if (moduleConfig.isReleaseManaged && env.BRANCH_NAME == "master") {
```

Comment:
Why no null check for the branch name here? Shouldn't this be a similar check to `isMaster(...)` as proposed above?

Reply:
Mixture of "way too deep into the rabbit warren, just want to finish!" and from this spot actually being null safe since `==` rather than `equals` :-)

Why is the _other_ place null checking and using `String.equals` ? Rabbit holes and 2 am coding ;-)